Directory.Build.props

```diff
@@ -52,6 +52,8 @@
   <ItemGroup Label="NuGet packages">
     <PackageReference Include="Newtonsoft.Json" Condition=" $(RequiresNewtonsoft)=='true'" Version="$(NewtonsoftVersion)" />
     <PackageReference Include="System.IO.Abstractions" Condition=" $(RequiresSystemIOAbstractions)=='true'" Version="9.0.4" />
+
+    <PackageReference Include="LibGit2Sharp" Version="0.26.2" Condition="$(RequiresLibGit)=='true'" GeneratePathProperty="true" PrivateAssets="all"  />  
```

Comment:
In theory, just adding the `LibGit2Sharp` NuGet package to the `ConnectedMode` project should have been enough for the VSIX build tools to recognise that the `LibGit2Sharp` package and the `LibGit2Sharp.NativeBinaries` it depends are required in the VSIX.

However, that wasn't working. The `LibGit2Sharp.NativeBinaries` package contents were added to the VSIX, but not the `LibGit2Sharp` managed package (even with the usual hacks in the `Integration.Vsix` project file about which output groups to include).

Instead, I marked the contents of the package with `PrivateAssets=all` and added the reference to the `ConnectedMode` project, its unit tests, and the `Integration .Vsix` project.

That causes the managed LibGit binaries and all of the native LibGit binaries (i.e. for all Unix platforms as well as Windows) to be added to the VSIX. This isn't exactly what we want, but it's a start and should unblock us.